### ISSUE TITLE
[Bug] onEnterViewFully causes an Uncaught TypeError

### ISSUE THREAD

**Issue by evanfrawley** [idyll-lang/idyll#259]
Including `onEnterViewFully` causes the idyll app to not even begin staying on a white screen with the following error in the console

error in console:

```
index.js:3328 Uncaught TypeError: watcher[_utils.scrollMonitorEvents[hook]] is not a function
    at index.js:3328
    at Array.forEach (<anonymous>)
    at index.js:3326
    at Array.forEach (<anonymous>)
    at IdyllDocument.initScrollListener (index.js:3320)
    at commitAttachRef (index.js:6877)
    at commitAllLifeCycles (index.js:7023)
    at HTMLUnknownElement.callCallback (index.js:5055)
    at Object.invokeGuardedCallbackDev (index.js:5071)
    at invokeGuardedCallback (index.js:5010)
```

Scroll events that work fine:
- `onEnterView`
- `onExitView`
- `onExitViewFully`

Here's the snippet of the code that I'm using it on:

```
[waypoint ]
[p onEnterViewFully:`sidePanelView = 1`]
Lorem ipsum dolor sit amet, consectetur adipiscing elit. Vivamus condimentum sapien quis eros cursus ultrices. Etiam dictum arcu vel elementum fermentum. Aenean ac consequat ex. Sed volutpat ipsum mollis urna ultricies tristique. Aliquam imperdiet sem quis sem aliquet, at maximus tellus fringilla. Pellentesque tristique, est eget porta condimentum, risus metus porttitor nulla, vitae tincidunt turpis magna a velit. Donec risus ex, hendrerit vel est quis, iaculis suscipit magna. Ut eleifend nulla id urna consequat scelerisque.
[/p]
[/waypoint]
```

Alternatively, I put the `onEnterViewFully` event in the `[waypoint]` component, but the app still crashed. 

Other debugging stuff I've done:
- made sure local & global `idyll` packages are on `2.1.6`
- used local `node_modules/.bin/idyll` binary to run locally
- built the project to see if there was a discrepancy between local & built, but there wasn't, still hit the same error

**Comment by mathisonian** [comment]
Thanks for catching this @evanfrawley. It looks like the crash is due to a typo, i've just made a PR to fix it (#260). I'm going to do some more thorough testing, then will release the update to npm.

**Comment by evanfrawley** [comment]
@mathisonian Thank you for the quick response!! FYI Using Idyll for the INFO 474 final project, so thank you for all of your work building out such a cool interactive tool!

**Comment by mathisonian** [comment]
No problem, It would be great to see what what you end up making! 

This is fixed in the latest release, `v2.1.7` on npm.
